src/styles/utils/_extends.scss

```diff
@@ -1,4 +1,99 @@
-%h1 {
-  font-family: "Roboto", sans-serif;
-  font-weight: 400;
+%bg-property {
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+%section-title {
+  position: relative;
+
+  margin-bottom: 53px;
+
+  text-align: center;
+  @include font(36px, bold, 36px);
+  letter-spacing: -1.5px;
+
+  @include desktop-screen {
+    margin-bottom: 93px;
+
+    @include font(48px, bold, 48px);
+  }
+
+  &::after {
+    content: "";
+
+    position: absolute;
+    top: calc(100% + 5px);
+    left: calc(50% - 23px / 2);
```

Comment:
Better to move these pixels to some meaningful variables because now it's not clear what are you calculating